tests/pipelines/eks/small/upstream-load.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: eks-small-upstream-load-pipeline-cron
+spec:
+  schedule: "0 8 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: curl
+            image: curlimages/curl
```

Comment:
alpine?

Reply:
AFASIU, `curlimages` image is alpine. check this out -  https://hub.docker.com/r/curlimages/curl

Reply:
Oh cool ok 👍